### DISCUSSION TITLE
图名放置在图件的正下方，中外文对照

### DISCUSSION THREAD

**Discussion by SilenceMonk** [Koyamin/ecnuthesis#11] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/72584051/232320784-744f74ba-6ff0-4388-b957-ac584227ae46.png)
请问中英文对照怎么修改代码实现哇？

**Comment by Koyamin** [comment, marked as answer]
https://github.com/Koyamin/ECNUThesis-Undergraduate/blob/7942dcbcdf3b00b7d1cee0b38884128af1784c2d/test.tex#L185-L189
请使用第188行的`\bicaption{中文}{English}`命令。
